transport_nantes/mobilito/templates/mobilito/session_summary.html

```diff
@@ -19,5 +31,61 @@ <h1>
             et {{ mobilito_session.public_transport_count|default:"0" }} transports collectifs.
         </p>
         
+        {% if request.user != mobilito_session.user.user %}
+            {% if user_has_reported_this_session %}
+                <span id="reported" class="small text-muted">
+                    Session signalée
```

Comment:
Can we please make this more subtle?

First, these aren't necessarily abuses, and we don't want to get into discussions about the circumstances under which something should be flagged.

Second, this steals attention.  It should be visible if you need it and peripheral otherwise.

This should a light-toned icon like a flag, which changes tone if already flagged.